db/migrate/20140805183007_create_carts.rb

```diff
@@ -1,5 +1,10 @@
 class CreateCarts < ActiveRecord::Migration
   def change
+    create_table :carts do |t|
+      t.integer :user_id
+      t.string :status, default: 'not submitted'
```

Comment:
This is absolutely not strictly necessary, but I would love to see `status` as an enum: http://edgeguides.rubyonrails.org/active_record_postgresql.html#enumerated-types